docs/podman-create.1.md

```diff
@@ -659,8 +659,7 @@ Note: Labeling can be disabled for all containers by setting label=false in the
 
 **--shm-size**=*size*
 
-Size of `/dev/shm`. The format is `<number><unit>`. `number` must be greater than `0`.
-Unit is optional and can be `b` (bytes), `k` (kilobytes), `m`(megabytes), or `g` (gigabytes).
+Size of `/dev/shm` (format: <number>[<unit>], where unit = b, k, m or g)
```

Comment:
Suggestion for considertation:
`b (bytes), k (kilobytes), m (megabytes), or g (gigabytes)`

I think most people know this, but just in case, it's nice to remove the question in their minds.

Reply:
That's better, thanks!